packages/accounts/src/testo.ts

```diff
@@ -0,0 +1,81 @@
+import { LocalAccountSigner, arbitrumSepolia } from "@alchemy/aa-core";
```

Comment:
Script used for local testing. You can try it yourself if you want! Will delete when moving out of draft status.